extensions/ipynb/src/helpers.ts

```diff
@@ -381,21 +386,29 @@ function convertOutputMimeToJupyterOutput(mime: string, value: Uint8Array) {
 		return '';
 	}
 	try {
-		const stringValue = new TextDecoder().decode(value.buffer.slice(value.byteOffset));
 		if (mime === CellOutputMimeTypes.error) {
+			const stringValue = textDecoder.decode(value.buffer.slice(value.byteOffset));
```

Comment:
Oops, the lines that call slice should just be passing the UIntArray, I will fix those when this is merged.